todo-backend/src/test/java/org/wildfly/quickstarts/todos/AbstractToDoIT.java

```diff
@@ -28,8 +28,8 @@
 import java.net.URL;
```

Comment:
We are missing the BasicRuntimeIT.java, which test code I believe should just be:

```
WebTarget client = ((ResteasyClientBuilder) ClientBuilder.newBuilder()).setFollowRedirects(true).build().target(getRequestUrl().toURI());
        GenericType<List<ToDo>> todosListType = new GenericType<List<ToDo>>() {
        };
        List<ToDo> allTodos = client.request().get(todosListType);
        assertEquals(0, allTodos.size());

```
IMHO we also I don't need an abstract class any longer, so perhaps merge AbstractToDoIT.java with RemoteToDoIT.java and rename it to ToDoIT.java, to serve as the quickstart specific testing?